### PR TITLE
Merge to develop

### PR DESCRIPTION
**Pull request by anwar6953**
I created this branch off develop.
Merged release/cdap-3.3-compatible into it.
Bumped the project version to the next snapshot.
Include guava dependency, since it was removed from cdap-api.

Build for kafka-pack: http://builds.cask.co/browse/CP-CCK2-3
Build for twitter-pack: http://builds.cask.co/browse/CP-CCT1-3

### CODE REVIEW

**Comment by anwar6953** [comment]
Started builds and added links to them above.

File: cdap-kafka-pack/cdap-kafka-flow/cdap-kafka-flow-compat-0.7/pom.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
   <parent>
     <artifactId>cdap-kafka-flow</artifactId>
     <groupId>co.cask.cdap</groupId>
-    <version>0.9.0-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>
```
Comment (bdmogal) [inline cdap-kafka-pack/cdap-kafka-flow/cdap-kafka-flow-compat-0.7/pom.xml]: should it be 0.10.0-SNAPSHOT? 1.0.0 represents a major release.

Comment (anwar6953) [inline cdap-kafka-pack/cdap-kafka-flow/cdap-kafka-flow-compat-0.7/pom.xml]: I can do that, yeah.

**Comment by bdmogal** [comment]
LGTM pending builds

**Pull request merged**